webpages/api/brainstorm/load_brainstorm.php

```diff
@@ -19,103 +21,104 @@ function convert_database_date_to_date($db_date) {
+	  AND d.brainstorm_support = 'Y'
+	ORDER BY d.display_order, d.divisionid, track_order;
 EOD;
    
-    $stmt = mysqli_prepare($db, $query);
-    if (mysqli_stmt_execute($stmt)) {
-        $result = mysqli_stmt_get_result($stmt);
-        $options = array();
-        $current_division = null;
-        while ($row = mysqli_fetch_object($result)) {
+	$stmt = mysqli_prepare($db, $query);
+	if (mysqli_stmt_execute($stmt)) {
+		$result = mysqli_stmt_get_result($stmt);
+		$options = array();
+		$current_division = null;
+		while ($row = mysqli_fetch_object($result)) {
 
-            if ($current_division == null || $current_division['id'] !== $row->divisionid) {
-                if ($current_division != null) {
-                    $options[] = $current_division;
-                }
-                $from_time = convert_database_date_to_date($row->from_time);
-                $to_time = convert_database_date_to_date($row->to_time);
-                $current_division = array(
-                    "id" => $row->divisionid,
-                    "name" => $row->divisionname,
-                    "from_time" => $from_time ? date_format($from_time, "c") : null,
-                    "to_time" => $to_time ? date_format($to_time, "c") : null,
-                    "tracks" => array()
-                );
-            }
-            $track = array(
-                "trackid" => $row->trackid,
-                "trackname" => $row->trackname
-            );
-            array_push($current_division['tracks'], $track);
-        }
-        mysqli_stmt_close($stmt);
-        if ($current_division != null) {
-            $options[] = $current_division;
-        }
-        return $options;
-    } else {
-        throw new DatabaseSqlException($query);
-    }
+			if ($current_division == null || $current_division['id'] !== $row->divisionid) {
+				if ($current_division != null) {
+					$options[] = $current_division;
+				}
+				$from_time = convert_database_date_to_date($row->from_time);
+				$to_time = convert_database_date_to_date($row->to_time);
+				$current_division = array(
+					"id" => $row->divisionid,
+					"name" => $row->divisionname,
+					"from_time" => $from_time ? date_format($from_time, "c") : null,
+					"to_time" => $to_time ? date_format($to_time, "c") : null,
+					"tracks" => array()
+				);
+			}
+			$track = array(
+				"trackid" => $row->trackid,
+				"trackname" => $row->trackname
+			);
+			array_push($current_division['tracks'], $track);
+		}
+		mysqli_stmt_close($stmt);
+		if ($current_division != null) {
+			$options[] = $current_division;
+		}
+		return $options;
+	} else {
+		throw new DatabaseSqlException($query);
+	}
 }
 
 function create_jwt_for_badgeid($db, $badgeid) {
-    $query = <<<EOD
+	$query = <<<EOD
  SELECT 
         P.password, P.data_retention, P.badgeid, C.firstname, C.lastname, C.badgename, C.regtype 
    FROM 
         Participants P 
    JOIN CongoDump C USING (badgeid)
   WHERE 
          P.badgeid = ?;
-EOD;
+ EOD;
```

Comment:
Just a note that prior to PHP 8.0 the EOD marker must be at the beginning of a line.
PHP 8.0+ allows the EOD marker to be indented (and the text in the block will be unindented by that amount).
Personally I think it's time to move to PHP 8, but if we want to continue to support PHP 7, this should be unindented.

Reply:
I thought it was the other way: I'm on PHP 7 and never hit a problem. I thought Leanne mentioned that PHP 8 became more persnickety about the indentation (I've tried to avoid indenting, but clearly I goofed, here).

Reply:
Addressed in f774f3e4e9edd65861e9caee3e47fab6464f139e